src/app/ApplicationTemplate.Shared/Presentation/Authentication/ForgotPasswordFormViewModel.cs

```diff
@@ -24,7 +25,11 @@ public class ForgotPasswordFormValidator : AbstractValidator<ForgotPasswordFormV
 	{
 		public ForgotPasswordFormValidator()
 		{
-			RuleFor(x => x.Email).NotEmpty().EmailAddress();
+#pragma warning disable CS0618 // EmailValidationMode.Net4xRegex validates for A@A.A. Default mode is only checking for A@A
```

Comment:
Use the non-deprecated alternative, even if it's not as good.

Reply:
@MatFillion told me otherwise

Reply:
We didn't want to create a custom mega regex for emails, but if we want just the simplier one it is ok.